CASC/PortfolioMode.cpp

```diff
@@ -224,80 +222,116 @@ bool PortfolioMode::performStrategy(Shell::Property* property)
  *
  * @author Giles
  **/
-void PortfolioMode::getExtraSchedules(Property& prop, Schedule& old, Schedule& extra, bool add_extra, int time_multiplier)
+void PortfolioMode::getExtraSchedules(Property& prop, Schedule& old, Schedule& extra, bool add_extra, float time_multiplier)
 {
   CALL("PortfolioMode::getExtraSchedules");
 
   // Add new extra_opts here
   Stack<vstring> extra_opts;
 
   if(add_extra){
+    /*
```

Comment:
More generally this fuzzing seem quite ad-hoc and some are only here because of the old schedules. Can some of these go? Can this whole function disappear?

Reply:
For instance, most of it could go from the point of view of the new snake schedules, but even these are already slowly getting outdated, so I think the function "as a concept" is nice to have! And that's for add_extra part.

The grow_time part is, in my opinion, generally useful. I can imagine a user with a lot of time and not much inspiration asking a schedule to run for days ... then it seems the best we can possibly offer is to repeat the strategies with larger and larger time/instruction limits.